stdlib/public/Python/Python.swift

```diff
@@ -1253,6 +1258,21 @@ extension PythonObject : MutableCollection {
   }
 }
 
+extension PythonObject: IteratorProtocol, Sequence {
```

Comment:
I believe the prevailing syntax is "ClassName : Protocol". Please update to that here and for MutableCollection.